sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -703,13 +729,18 @@ async def test_async_queue_by_servicebus_client_fail_send_messages(self, service
             async with sb_client.get_queue_sender(servicebus_queue.name) as sender:
                 with pytest.raises(MessageSendFailed):
                     await sender.send(Message(too_large))
-
-            async with sb_client.get_queue_sender(servicebus_queue.name) as sender:
-                sender.queue_message(Message(too_large))
-                results = await sender.send_pending_messages()
-                assert len(results) == 1
-                assert not results[0][0]
-                assert isinstance(results[0][1], MessageSendFailed)
+                    
+                half_too_large = "A" * int((1024 * 512) / 2)
+                with pytest.raises(ValueError):
+                    await sender.send([Message(half_too_large), Message(half_too_large)])
```

Comment:
(would this raise ValueError out? I thought the max messages size supported by service is 1024 * 1024 bytes if it's a standard tier SB)

Reply:
So yes, it will throw, (had run the tests locally) but your question prompted me to sanity check the size the test was using initially and it turns out it might be TOO large; per docs here https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quotas
 the answer is "256KB", so I'll use this knowledge to make the test as a whole more targeted.